src/components/Avatar/style.css

```diff
@@ -0,0 +1,77 @@
+.avatar-base {
+  height: 100%;
+  width: 100%;
+  border-radius: 100% !important;
+  object-fit: cover !important;
+}
+
+.blank_large {
```

Comment:
Would suggest keeping class names consistent for the ease of the user. If you are using a hyphen for naming your classes, keep that going.